case-studies/avinya-foundation.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: ballerina-publishing-to-central-left-nav-pages-swanlake
```

Comment:
Is this layout correct?

Reply:
Actually we don't need it. @praneesha you can remove them.

Reply:
Removed the layout from all the case study pages as it is not required for JS pages.